packages/video_player_videohole/tizen/src/messages.cc

```diff
@@ -0,0 +1,1140 @@
+                                              WrapError(output.value())));
+              } else {
+                wrapped.insert(std::make_pair(flutter::EncodableValue("result"),
+                                              flutter::EncodableValue()));
+              }
+            } catch (const std::exception& exception) {
+              wrapped.insert(std::make_pair(flutter::EncodableValue("error"),
+                                            WrapError(exception.what())));
+            }
+            reply(flutter::EncodableValue(wrapped));
+          });
+    } else {
+      channel->SetMessageHandler(nullptr);
+    }
+  }
+  {
+    auto channel =
+        std::make_unique<flutter::BasicMessageChannel<flutter::EncodableValue>>(
+            binary_messenger, "dev.flutter.pigeon.VideoPlayerApi.setDisplayRoi",
+            &GetCodec());
+    if (api != nullptr) {
+      channel->SetMessageHandler(
+          [api](const flutter::EncodableValue& message,
+                const flutter::MessageReply<flutter::EncodableValue>& reply) {
+            flutter::EncodableMap wrapped;
+            try {
+              auto args = std::get<flutter::EncodableList>(message);
+              auto encodable_arg_arg = args.at(0);
+              if (encodable_arg_arg.IsNull()) {
+                wrapped.insert(
+                    std::make_pair(flutter::EncodableValue("error"),
+                                   WrapError("arg_arg unexpectedly null.")));
+                reply(flutter::EncodableValue(wrapped));
+                return;
+              }
+              const GeometryMessage& arg_arg =
+                  std::any_cast<const GeometryMessage&>(
+                      std::get<flutter::CustomEncodableValue>(
+                          encodable_arg_arg));
+              std::optional<FlutterError> output = api->SetDisplayRoi(arg_arg);
+              if (output.has_value()) {
+                wrapped.insert(std::make_pair(flutter::EncodableValue("error"),
+                                              WrapError(output.value())));
+              } else {
+                wrapped.insert(std::make_pair(flutter::EncodableValue("result"),
+                                              flutter::EncodableValue()));
+              }
+            } catch (const std::exception& exception) {
+              wrapped.insert(std::make_pair(flutter::EncodableValue("error"),
+                                            WrapError(exception.what())));
+            }
+            reply(flutter::EncodableValue(wrapped));
+          });
+    } else {
+      channel->SetMessageHandler(nullptr);
+    }
+  }
+  {
+    auto channel =
+        std::make_unique<flutter::BasicMessageChannel<flutter::EncodableValue>>(
+            binary_messenger,
+            "dev.flutter.pigeon.VideoPlayerApi.setBufferingConfig",
+            &GetCodec());
+    if (api != nullptr) {
+      channel->SetMessageHandler(
+          [api](const flutter::EncodableValue& message,
```

Comment:
`api` is captured but not being used.

Reply:
> `api` is captured but not being used.

These codes in 'messages.cc' , 'messages.h' and 'messages.dart' are autogenerated from Pigeon (v3.2.0), better not edit directly.